core/src/main/java/org/apache/accumulo/fate/ReadOnlyTStore.java

```diff
@@ -139,4 +139,12 @@
    */
   List<Long> list();
 
+  /**
+   * Retrieve timestamp of a transaction.
+   *
+   * @param tid
+   *          Transaction id, previously reserved.
+   * @return timestamp of transaction.
+   */
+  String timeTopCreated(long tid);
```

Comment:
I'm just confused on the `Top` portion of the name. What does that mean?

Reply:
Table operation, I believe. Some of the other functions in that class use that name. I do agree it could be confusing. `timeCreated` may just be better.

Reply:
TOP is the operation that created the operation stack. I'm not sure if it is table operation or not, but FATE operations create a stack of operations and top represents the parent (I think)  This is coming from someone that took an embarrassing long time to realize that REPO stood for REPeatable Operation and not just generic repository. So...

Reply:
Oh, that makes sense. I will go ahead and change the function names to avoid any confusion in the future.